docs/docsite/rst/userguide/inventories.rst

```diff
@@ -726,6 +753,7 @@ This opens the Details tab of the selected host.
 8. If a host was used to run a job, you can view details about those jobs in the **Completed Jobs** tab of the host and click **Expanded** to view details about each job.
 
 .. image:: ../common/images/inventories-add-host-view-completed-jobs.png
+   :alt: Inventories add host view completed jobs
```

Comment:
Suggested wording: "Jobs tab showing list of completed jobs associated with the selected host"